### PR TITLE
Fixes Druid Profiler failures

### PR DESCRIPTION
**Pull request by TeddyCr**
### Describe your changes:
- implement fixes for druid profiler workflow
- stabilize playwright tests
- implement redshift, druid, hive playwright tests

I worked on ... because ...

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: .github/workflows/playwright-integration-tests-mysql.yml [hunk 1]
```diff
@@ -14,17 +14,18 @@
 
 name: MySQL Playwright Integration Tests
 on:
+  pull_request:
```
Comment (akash-jain-10) [inline .github/workflows/playwright-integration-tests-mysql.yml:17]: Hey @TeddyCr  - Do we need any constraints here on which file changes to run this CI or which paths to ignore ?

File: .github/workflows/playwright-integration-tests-postgres.yml [hunk 2]
```diff
@@ -14,17 +14,18 @@
 
 name: Postgres Playwright Integration Tests
 on:
+  pull_request:
```
Comment (akash-jain-10) [inline .github/workflows/playwright-integration-tests-postgres.yml:17]: Hey @TeddyCr - Do we need any constraints here on which file changes to run this CI or which paths to ignore ?

File: .github/workflows/playwright-integration-tests-mysql.yml (hunk 1 above)
Comment (TeddyCr) [reply .github/workflows/playwright-integration-tests-mysql.yml:17]: yep, I just set it like that to test it on this PR 😊

File: openmetadata-docs/content/v1.2.x-SNAPSHOT/connectors/ingestion/workflows/profiler/index.md [hunk 3]
```diff
@@ -67,6 +67,8 @@ Set the sample to be use by the profiler for the specific table.
 - `Percentage`: Value must be between 0 and 100 exclusive (0 < percentage < 100). This will sample the table based on a percentage
 - `Row Count`: The table will be sampled based on a number of rows (i.e. `1,000`, `2,000`), etc.
 
+⚠️ This option is currently not support for Druid. Sampling leverage `RANDOOM` functions in most database (some have specific sampling functions) and Druid provides neither of these option. We recommend using the partitionning or sample query option if you need to limit the amount of data scanned.
```
Comment (pmbrull) [inline openmetadata-docs/content/v1.2.x-SNAPSHOT/connectors/ingestion/workflows/profiler/index.md:70]:
```suggestion
⚠️ This option is currently not support for Druid. Sampling leverage `RANDOM` functions in most database (some have specific sampling functions) and Druid provides neither of these option. We recommend using the partitionning or sample query option if you need to limit the amount of data scanned.
```

File: openmetadata-docs/content/v1.2.x-SNAPSHOT/connectors/ingestion/workflows/profiler/index.md [hunk 4]
```diff
@@ -107,6 +109,8 @@ Set the sample to be use by the profiler for the specific table.
...
```
Comment (pmbrull) [inline openmetadata-docs/content/v1.2.x-SNAPSHOT/connectors/ingestion/workflows/profiler/index.md:112]:
```suggestion
⚠️ This option is currently not support for Druid. Sampling leverage `RANDOM` functions in most database (some have specific sampling functions) and Druid provides neither of these option. We recommend using the partitionning or sample query option if you need to limit the amount of data scanned.
```

**Review by pmbrull — DISMISSED** [review]

**Review by akash-jain-10 — APPROVED** [review]

File: ingestion/tests/e2e/entity/database/test_redshift.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+                        metadata=IngestionTestConfig(
+                            schema_=IngestionFilterConfig(includes=["dbt_jaffle"]),
+                        ),  # type: ignore
+                        profiler=IngestionTestConfig(
+                            table=IngestionFilterConfig(includes=["customers"]),
+                        ),  # type: ignore
+                    ),
+                    validation=ConnectorValidationTestConfig(
+                        profiler=ValidationTestConfig(
+                            database="dev", schema_="dbt_jaffle", table="customers"
```
Comment (TeddyCr) [inline ingestion/tests/e2e/entity/database/test_redshift.py:43]:
```suggestion
                            database="e2e_cli_tests", schema_="dbt_jaffle", table="customers"
```

**Review by pmbrull — APPROVED** [review]

**Pull request merged**